csharp/Ice/Greeter/README.md

```diff
@@ -2,7 +2,7 @@
 
 The Greeter demo illustrates how to send a request and wait for the response.
 
-This demo provides two implementations on the server: a synchronous dispatch implementation (in Server), and an
+This demo provides two implementations for the server: a synchronous dispatch implementation (in Server), and an
```

Comment:
I think it's fine to leave the wording alone here, since we say "this demo" immediately on the next line.